types/node/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "private": true,
+    "types": "index",
+    "typesVersions": {
+        ">4.9.0-a": { "*": ["*"] },
+        "<=4.8": { "*": ["ts4.8/*"] }
```

Comment:
```suggestion
        ">=4.9.0-0": { "*": ["*"] },
        "<4.9.0-0": { "*": ["ts4.8/*"] }
```

Reply:
Actually, with `"<4.9.0-0"`, `">=4.9.0-0"` is no longer needed. I'll change dtslint to allow `<=4.8` or `<4.9.0-0`